CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 * None
 
 ### Fixed
-* None
+* Fixed an issue that could cause errors or loops with client reset on certain platforms. (PR [2830](https://github.com/realm/realm-dotnet/pull/2830))
```

Comment:
Shouldn't it mention what platforms?

Reply:
I did not write it because there was a different behaviour on iOS/Android/Windows and didn't want to get too much into details.

Reply:
I understand that the bug created different behaviours (errors) on different platforms. In fact, not being specific like e.g. "*error Y* on iOS and *error X* on Windows" is very fine in my opinion.
But I still see value for our users to be able to quickly read something like "fixed issue A and B that we sometimes saw happening on platform X and Y".
Just my opinion.

Reply:
I think it's fine to just leave out the `certain platforms` altogether. It was a bug and even if it worked by accident on some platforms, that's probably not worth calling out.